elasticsearch/utils/es_seed_acl

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+source "logging"
+
+MAX_WAIT_SEED=${MAX_WAIT_SEED:-$((60*60*24))}
+
+info "Seeding the searchguard ACL index"
+now=0
+while ! /usr/share/elasticsearch/plugins/search-guard-2/tools/sgadmin.sh \
+    -cd ${HOME}/sgconfig \
+    -i .searchguard.${DC_NAME} \
+    -ks /etc/elasticsearch/secret/searchguard.key \
+    -kst JKS \
+    -kspass kspass \
+    -ts /etc/elasticsearch/secret/searchguard.truststore \
+    -tst JKS \
+    -tspass tspass \
+    -nhnv \
+    -icl ${DEBUG:+-dg} && [ ${now} -lt ${MAX_WAIT_SEED} ]
+do
+    warn "Error seeding the searchguard ACL index... retrying in 10 seconds"
+    warn "Seeding will continue to fail until the cluster status is YELLOW"
+    info "Remaining red indicies: $(QUERY=_cat/indicies es_util | grep "^red" | wc -l)"
+    sleep 10
+    now=$((now+40)) #wait plus 30s timeout from sgadmin
```

Comment:
The timeout is 30 seconds from sgadmin?  Then we will get this error message every 40 seconds?  That's an awful lot of spam if sgadmin is having problems.  Change it so that it only prints the message every hour or so, and print the number of retry attempts made in that interval.